project2/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Overpass:400,700|Questrial|Righteous&display=swap"
+        rel="stylesheet">
+    <script src="https://kit.fontawesome.com/f7ec3810d9.js" crossorigin="anonymous"></script>
+    <title>MOONVIES</title>
+</head>
+
+<body>
+    <h1>MOONVIES</h1>
+    <nav class="nav">
+        <ul>
+            <li><a href="movienav">MOVIES</a></li>
```

Comment:
Here and on line 16, you want to control capitalization via CSS and `text-transform`, not by all caps markup.